usl_models/tests/flood_ml/mock_dataset.py

```diff
@@ -11,11 +11,13 @@ def input_signature(
     params: model_params.FloodModelParams,
     height: int = 100,
     width: int = 100,
-    n: int = 0,
+    n: int | None = None,
```

Comment:
I think it'd be helpful to use a more descriptive name than 'n' here. Here it looks like the rainfall duration, while the 'n' below in label_signature looks like the batch size, and the 'n' in mock_dataset is rainfall duration.

Also, I think the rest of our code is doing `Optional[X]` rather than `X | None` so would like to stay consistent

Reply:
In this file `n` always represents the rainfall duration, which should be clear from docstrings. It's also consistent with the `n` from `call_n`.

We already have `X | None` in the codebase and this is the "modern" 3.10 approach so I prefer to migrate the rest of the codebase at a later time.